k8s/upgrade-job/src/opts.rs

```diff
@@ -41,10 +41,22 @@ pub(crate) struct CliArgs {
     )]
     core_chart_dir: Option<PathBuf>,
 
+    /// This is the path to upgrade exception file.
+    #[arg(
+        long,
+        env = "UPGRADE_EXCEPTION_FILE_PATH",
+        default_value = "/k8s/upgrade/config/unsupported_versions.yaml"
```

Comment:
nit: default_value is left for backwards compatibility. Better to use default_value_t.

This value should be in the constants file... and the constant should be used here.